controllers/users.js

```diff
@@ -762,6 +762,21 @@ async function usersPatchHandler(req, res) {
   }
 }
 
+const updateUsernamesToLowercase = async (req, res) => {
```

Comment:
the data variable returned from the updateUsernameToLowercase() function is not being checked for errors before it is passed to the res.json() function. This means that if the updateUsernameToLowercase() function fails, the data variable may contain an error message, which will be returned to the client.